src/events/MessageScanner.event.ts

```diff
@@ -0,0 +1,170 @@
+import { Message } from "discord.js";
+import { Event } from "../handlers/EventHandler";
+import Tesseract from "tesseract.js";
+import fetch from "node-fetch";
+import { client } from "../index";
+
+export default class ReadyEvent extends Event<"messageCreate"> {
+    public event = "messageCreate";
+    public async run(msg: Message) {
+        if (
+            !msg.guild ||
```

Comment:
can the `||` be at the start of the new line instead of the previous line? makes it more readable